### ISSUE TITLE
Running two emulators

### ISSUE THREAD

**Issue by 2byrds** [roots-id/rootswallet#114]
When trying to run two RootsWallet Android Emulators we were seeing:
![image](https://user-images.githubusercontent.com/681493/193811730-1edd5016-69da-4d36-948c-b933cb65b00e.png)

**Comment by 2byrds** [comment]
MeGrimLance — Yesterday at 8:10 PM
interesting.  it's been a long time since i tried two emulators but remember getting errors and never pursued it.  i suppose you are connecting to the same metro server/instance but your storage is empty, etc.... not sure exactly tho
[8:11 PM]
are you using the same avd for both?

AlexA — Yesterday at 8:12 PM
I am using the same avd, just made a copy and after did an expo start I did a shit+a and then switched emulators
[8:12 PM]
I also tried doing npx react-native run-android --deviceId=emulator-5556 --no-jetifier but the build fails

[8:37 PM]
managed to get two of them running expo run:android --device=Copy_of_rootswallet
